plugins/pagerduty/README.md

```diff
@@ -45,6 +45,18 @@ import {
 }
 ```
 
+Add the proxy configuration in `app-config.yaml`
+
+```yaml
+proxy:
+  ...
+  '/pagerduty':
+    target: https://api.pagerduty.com
+    headers:
+      Authorization:
+        $env: PAGERDUTY_TOKEN
```

Comment:
Let's use the new env var substitution instead of requiring the somewhat awkward format in the env var itself.

```yaml
Authorization: Token token=${PAGERDUTY_TOKEN}
```

I think it's best to also merge the "Providing the API Token" section below into this one.